Sources/Purchasing/PurchasesOrchestrator.swift

```diff
@@ -302,9 +302,8 @@ class PurchasesOrchestrator {
                   package: Package?,
                   completion: @escaping PurchaseCompletedBlock) {
         guard let productIdentifier = sk1Product.extractProductIdentifier(withPayment: payment) else {
-            Logger.error(Strings.purchase.could_not_purchase_product_id_not_found)
```

Comment:
Creating the error already logs it, so this was duplicated. The new code relies on the `String` without hardcoding another error message.